fj-21/avanzados/sesion1.html

```diff
@@ -0,0 +1,133 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>PUMM | Programando un Mexico Mejor </title>
+  <!--favicon, imagen en el title-->
+  <link rel="shortcut icon" href="../favicon.ico">
+  <!-- Bootstrap core CSS -->
+  <link href="../libraries/dist/css/bootstrap.min.css" rel="stylesheet">
+
+  <!-- Custom styles for this template -->
+  <link href="../styles.css" rel="stylesheet">
+</head>
+
+<body>
+  <nav class="navbar sticky-top navbar-dark" style="background: rgb(0,172,228);
+      background: linear-gradient(65deg, rgba(0,172,228,1) 0%, rgba(23,12,102,1) 100%);">
+    <div class="container d-flex justify-content-between">
+      <a href="../index.html" class="navbar-brand">
+        <!--<img src="./libraries/assets/pumm-logo/saitc.png" alt="PUMM logo" height="40" class="d-inline-block align-middle mr-2">-->
+        <span>PUMM</span>
+        Programando un México Mejor
+      </a>
+      <div>
+        <a href="../about.html" class="nav-link d-inline-block" style="color: white;">Acerca del proyecto</a>
+        <a href="../equipo.html" class="nav-link d-inline-block" style="color: white;">Equipo</a>
+        <a href="../clases.html" class="nav-link d-inline-block" style="color: white;">Clases</a>
+        <a href="../recursos.html" class="nav-link d-inline-block" style="color: white;">Recursos</a>
+        <a href="../proximamente.html" class="nav-link d-inline-block" style="color: white;">Blog</a>
+      </div>
+    </div>
+  </nav>
+
+  <div id="resource-section" class="album">
+    <div class="container">
+      <h1 class="text-center pb-3">Introducción a HTML y Etiquetas básicas - Parte 1</h1>
+      <div class="white-bg row py-4">
+        <div class="col-12 col-md-7 col-12 text-center">
+          <h2>¿De qué trata la sesión?</h2>
+          <p class="lead text-justify">HTML es un lenguaje de marcado para la elaboración de páginas web. Un estándar
+            que sirve de referencia del software que conecta con la elaboración de páginas web en sus diferentes
+            versiones, define una estructura básica y un código (denominado código HTML) para la definición de contenido
+            de una página web, como texto, imágenes, videos, juegos, entre otros.</p>
+          <p class="lead">Encuentra el manual de instalación de Atom, el editor de texto que estaremos utilizando, dando
```

Comment:
No utilizaremos Atom, es repl.it. Si gustan agregar la guía de uso que creamos estaría perfecto. La pueden encontrar en el drive